quinn-proto/src/endpoint.rs

```diff
@@ -1052,6 +1050,23 @@ pub enum ConnectError {
     UnsupportedVersion,
 }
 
+/// Error for attempting to retry an [`IncomingConnection`] that can not be retried
```

Comment:
Let's be as specific and actionable as possible: s/can not be retried/whose address was already validated by a previous retry/ or similar.

Reply:
Both rephrased to:

```
/// Error for attempting to retry an [`IncomingConnection`] which already bears an address
/// validation token from a previous retry
```